test/unit/bucket/test_bucket.py

```diff
@@ -1480,6 +1480,16 @@ def test_upload_local_file_incremental(self):
 
                 last_data = data
 
+    def assertBucketContents(self, expected, *args, **kwargs):
```

Comment:
This function is unused despite being just added. Did you mean to use it instead of `ls` function in `test_custom_timestamp`?

Reply:
good point